packages/neon/neon/test/timer_bloc_test.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:neon/neon.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('TimerBloc', () {
```

Comment:
As mentioned above we should test that dispose stops all timers and clears it's underlying data.
Being a singleton we should verify this.